docs/api-reference/webgl/transform-feedback.md

```diff
@@ -31,6 +31,8 @@ const program = new Program(gl, {
   varyings: ['gl_Position'],
 });
 
+[//]: # transformFeedback is not defined yet
```

Comment:
no, it was a note that we did not declare transformFeedback before using it in the doc.
will remove for this PR.